### PR TITLE
Refreshed dates

### PR DESCRIPTION
**Pull request by xypnox**
Added dates for the spring semester of 2018-19. Tested with identical gyft2 also.

### CODE REVIEW

File: dates.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
 import sys
 
 # SEM_BEGIN=datetime.datetime.now(pytz.timezone('Asia/Kolkata'))
-SEM_BEGIN=build_event.generateIndiaTime(2018, 7, 17, 0, 0)
+SEM_BEGIN = build_event.generateIndiaTime(2019, 1, 1, 0, 0)
```
Comment (thealphadollar) [inline dates.py]: This should be 2, I think.

File: dates.py [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
 
-MID_TERM_BEGIN=build_event.generateIndiaTime(2018, 9, 17, 0, 0)
+MID_TERM_BEGIN = build_event.generateIndiaTime(2019, 2, 17, 0, 0)
```
Comment (thealphadollar) [inline dates.py]: Should be 2019,2,18

File: dates.py [hunk 3]
```diff
@@ -6,13 +6,13 @@
...
 
-MID_TERM_END=build_event.generateIndiaTime(2018, 9, 26, 23, 59)
+MID_TERM_END = build_event.generateIndiaTime(2019, 2, 26, 23, 59)
 
-END_TERM_BEGIN=build_event.generateIndiaTime(2018, 11, 16, 0, 0)
+END_TERM_BEGIN = build_event.generateIndiaTime(2019, 4, 21, 0, 0)
```
Comment (thealphadollar) [inline dates.py]: SHould be 2019,4,22

File: dates.py (hunk 1 above)
Comment (xypnox) [reply dates.py]: This is a bug with python. This happens for all the starting dates. If we put two, the events start from 3 etc.

**Comment by xypnox** [comment]
Please check the dates once. The dates are _technically_wrong but they do produce the correct calendar.

**Comment by xypnox** [comment]
Sure 👍

**Pull request merged**